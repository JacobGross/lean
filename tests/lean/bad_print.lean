constant boo : nat

print definition boo

print 2

print fields nat
