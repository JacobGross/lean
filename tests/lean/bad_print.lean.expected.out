bad_print.lean:3:17: error: invalid 'print definition', 'boo' is not a definition
bad_print.lean:5:6: error: invalid print command
bad_print.lean:7:13: error: invalid 'print fields' command, 'nat' is not a structure
